### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by MartinHolmes82** [dranger003/WTSPlugin1#3]
This looks very interesting. We are currently looking at creating a virtual channel ourselves. Do you have any instructions on using this and its pointers?

**Comment by patv2learn** [comment]
> This looks very interesting. We are currently looking at creating a virtual channel ourselves. Do you have any instructions on using this and its pointers?

Yes, this is an interesting approach I have tried to work on this for one of my school project where I got able to develop server to client communication but was not able to finish client to server communication.

I could not finish that project due to client to server communication but really interested to learn if there is anything on this. Please let me know if you need any input about server to client part, I may have to brush-up but did spent good amount of time on this.

Thanks.
